syntax = "proto3";
package message;

message GetNameReply {
    uint64 protocol_id = 1;
    uint64 cmd_id = 2;
    string name = 3;
}